test/Amazon.Lambda.Tools.Test/DeployTest.cs

```diff
@@ -190,7 +190,7 @@ public async Task RunDeploymentNativeAot(bool multipleProjects, bool customMount
         /// This test deploys a .NET8 Native AOT Web App that does not explicitly set the OutputType to Exe.
         /// In .NET8, the web app inherits the OutputType from the SDK.
         /// </summary>
-        [Fact]
+        [Fact(Skip = "This test requires Docker to be available. We are using a Windows container in the release pipeline which does not support Docker.")]
```

Comment:
Does the same check we use above apply here? I'd rather leave it on-by-default for developers working locally if possible.

https://github.com/aws/aws-extensions-for-dotnet-cli/blob/ea50d25688398071096d7a574eb814a2fedfce3c/test/Amazon.Lambda.Tools.Test/DeployTest.cs#L45-L49

Reply:
Fixed